### PR TITLE
Reforger

### PR DESCRIPTION
**Pull request by granQ**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,29 @@
...
+      A new era of SFP. All brand new content, the only legancy is the name and the ambition to bring the most of Swedish armed forces into Arma.
+      Our first release covered the basic infantry squad of 1989-1992 era and much more content is planned.  
+    </p>
+
+    <p>
+      <a class="btn btn-secondary btn-lg" href="/reforger/">
+        <span class="fa fa-info-circle"></span>
+        More Info
+      </a>
+      <a class="btn btn-primary btn-lg" href="/reforger/">
```
Comment (Dahlgren) [inline index.html]: Same link as above? 🤔

File: _includes/navigation.html [hunk 2]
```diff
@@ -4,6 +4,11 @@
       <span class="navbar-toggler-icon"></span>
     </button>
     <a class="navbar-brand" href="/">Swedish Forces Pack</a>
+    
+        <li class="nav-item">
+          <a class="nav-link" href="/reforger">Reforger</a>
+       </li>
```
Comment (Dahlgren) [inline _includes/navigation.html]: Fix indentation
```suggestion
    <li class="nav-item">
      <a class="nav-link" href="/reforger">Reforger</a>
    </li>
```

File: reforger/index.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+<div class="container">
+  <h1>Swedish Forces Pack</h1>
+</div>
+
+<div class="container">
+  <h2>Description</h2>
+
+  <p>
+   The original Swedish Armed Forces experience for ArmA are now entering the next generation with the first release for Arma Reforger.
```
Comment (Dahlgren) [inline reforger/index.html]: Fix indentation. It's Arma, not ArmA
```suggestion
    The original Swedish Armed Forces experience for Arma are now entering the next generation with the first release for Arma Reforger.
```

File: reforger/index.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  </p>
+
+  <p>
+    SFP is now leaving the previous games and relases behind and continue with 100% new content made for Reforger with professional tools with the highest quality. <br/>
+    Focus is infantry and the era (1988-1992) represented in Reforger. 
+  </p>
+
+  <p>
+   Our first release comes with a Conflict version and a coop in order to test our mod, but more missions are planned, both cooperative and competitive with the hopes to bring players from ArmA3 to Reforger as a preperation for ArmA4.
```
Comment (Dahlgren) [inline reforger/index.html]: Fix indentation. Arma, not ArmA
```suggestion
    Our first release comes with a Conflict version and a coop in order to test our mod, but more missions are planned, both cooperative and competitive with the hopes to bring players from Arma 3 to Reforger as a preperation for Arma 4. 
```

File: reforger/index.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+<div class="container">
+  <h2>Description</h2>
+
+  <p>
+   The original Swedish Armed Forces experience for ArmA are now entering the next generation with the first release for Arma Reforger.
+  </p>
+
+  <p>
+    SFP is now leaving the previous games and relases behind and continue with 100% new content made for Reforger with professional tools with the highest quality. <br/>
```
Comment (Dahlgren) [inline reforger/index.html]:
```suggestion
    SFP is now leaving the previous games and releases behind and continue with 100% new content made for Reforger with professional tools with the highest quality. <br/>
```

File: reforger/index.html [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+      </ul>
+    </div>
+   
+    <div>
+      <h4>Rifles</h4>
+      <ul>
+        <li>Ak4 - the iconic H&amp;K G3A3 modified for Swedish use. The versions include with M203 attachment and marksman edition..</li>
+        <li>Ak5 - the FN FNC, standard version and the Bertil (Marksman with SUSAT optics)  </li>
+        <li>Kpist m/45 - the "Swedish K", a fully automatic submachinegun used by the Swedish Armed Forces between 1945 and 2007.</li>
+         <li>KSP 58B - the FN MAG.</li>
```
Comment (Dahlgren) [inline reforger/index.html]: Fix indentation
```suggestion
        <li>KSP 58B - the FN MAG.</li>
```

**Review by Dahlgren — CHANGES_REQUESTED** [review]
The PNG images are **HUGE**, it won't be a pleasant experience for the page loading. Use JPG for screenshots.

File: index.html (hunk 1 above)
Comment (granQ) [reply index.html]: Just wanted people to understand it is out.. but then its not a download. Will remove.

**Review by Dahlgren — APPROVED** [review]

**Pull request merged**